tools/ops/azure/container-host/aci/aci-template.json

```diff
@@ -0,0 +1,111 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aci_name": {
+            "defaultValue": "custodian-aci",
```

Comment:
nit: can we append subscription id here? (so we can deploy multiple ACIs without specifying this parameter.